plugin/grpc/grpcstats/client_metrics.go

```diff
@@ -149,142 +125,6 @@ func defaultClientViews() {
 		RPCClientResponseCountView,
 	)
 	// TODO(jbd): Add roundtrip_latency, uncompressed_request_bytes, uncompressed_response_bytes, request_count, response_count.
-
-	RPCClientRoundTripLatencyMinuteView, _ = stats.NewView(
```

Comment:
What's the rationale behind us getting rid of non-cumulative views?

Reply:
https://github.com/census-instrumentation/opencensus-specs/issues/47

Reply:
Thanks for the link @ramonza.